boards/deneyapkart.json

```diff
@@ -7,19 +7,20 @@
     "extra_flags": [
       "-DARDUINO_DYDK",
       "-DBOARD_HAS_PSRAM",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1",
       "-mfix-esp32-psram-cache-issue",
       "-mfix-esp32-psram-cache-strategy=memw"
     ],
     "f_cpu": "240000000L",
     "f_flash": "80000000L",
-    "flash_mode": "qio",
+    "flash_mode": "dio",
```

Comment:
Could you please explain the reasoning behind changing the `flash_mode` option? It seems `QIO` is used when selecting bootloader for the Deneyap boards in the Arduino core, is it wrong here https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L17358 ?

Reply:
> Could you please explain the reasoning behind changing the `flash_mode` option? It seems `QIO` is used when selecting bootloader for the Deneyap boards in the Arduino core, is it wrong here https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L17358 ?

I think that "flash_mode" parameter is equivalent at https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L17357. 
If it is the equivalent of the "boot" parameter of arduino-esp32 repo, I have to update it here.

Reply:
> If it is the equivalent of the "boot" parameter of arduino-esp32 repo, I have to update it here.

Yes, it's equivalent, so I'd recommend using bootmode value from boards.txt in this field. The transformation to a proper `flash_mode` happens here https://github.com/platformio/platform-espressif32/blob/develop/builder/main.py#L68

Reply:
Update all `flash_mode` parameter.